internal/events/dx_callbacks.go

```diff
@@ -175,15 +175,15 @@ func (em *eventManager) markUnpinnedMessagesConfirmed(ctx context.Context, batch
 	return nil
 }
 
-func (em *eventManager) DXEvent(dx dataexchange.Plugin, event dataexchange.DXEvent) {
+func (em *eventManager) DXEvent(ctx context.Context, dx dataexchange.Plugin, event dataexchange.DXEvent) {
 	switch event.Type() {
 	case dataexchange.DXEventTypePrivateBlobReceived:
 		em.privateBlobReceived(dx, event)
 	case dataexchange.DXEventTypeMessageReceived:
 		// Batches are significant items of work in their own right, so get dispatched to their own routines
 		go em.messageReceived(dx, event)
 	default:
-		log.L(em.ctx).Errorf("Invalid DX event type from %s: %d", dx.Name(), event.Type())
+		log.L(em.ctx).Errorf("Invalid data exchange event type from %s: %d", dx.Name(), event.Type())
```

Comment:
if we're now passing in a context, should we use that one for logging?

Reply:
I started to do this, but it proved to be a fairly large change to the way the event handling (and particularly the unit tests) are laid out. I altered the commit to not pass the context, since there's no compelling reason we need to use it right now vs. the context on event manager.